src/MongoDB.Driver.Legacy/MongoServerSettings.cs

```diff
@@ -110,7 +110,7 @@ public MongoServerSettings()
             _servers = new List<MongoServerAddress> { new MongoServerAddress("localhost") };
             _serverSelectionTimeout = MongoDefaults.ServerSelectionTimeout;
             _socketTimeout = MongoDefaults.SocketTimeout;
-            _sslSettings = null;
+            _sslSettings = new SslSettings();
```

Comment:
Changing the default value from null to not null seems like a potentially breaking change.

Do you really need to do this?